tests/macros/test-WebAssemblySidebar.js

```diff
@@ -15,30 +15,36 @@ var expected = `\
 
 <ol>
   <li data-default-state="open"><a href="/en-US/docs/WebAssembly"><strong>WebAssembly home page</strong></a>
-  <li data-default-state="open"><a href="/en-US/docs/WebAssembly">Tutorials</a>
-    <ol>
-      <li><a href="/en-US/docs/WebAssembly/Concepts">WebAssembly concepts</a></li>
-      <li><a href="/en-US/docs/WebAssembly/C_to_wasm">Compiling from C/C++ to WebAssembly</a></li>
-      <li><a href="/en-US/docs/WebAssembly/Using_the_JavaScript_API">Using the WebAssembly JavaScript API</a></li>
-      <li><a href="/en-US/docs/WebAssembly/Understanding_the_text_format">Understanding WebAssembly text format</a></li>
-      <li><a href="/en-US/docs/WebAssembly/Text_format_to_wasm">Converting WebAssembly text format to wasm</a></li>
-      <li><a href="/en-US/docs/WebAssembly/Loading_and_running">Loading and running WebAssembly code</a></li>
-      <li><a href="/en-US/docs/WebAssembly/Caching_modules">Caching compiled WebAssembly modules</a></li>
-      <li><a href="/en-US/docs/WebAssembly/Exported_functions">Exported WebAssembly functions</a></li>
-    </ol>
+  <li class="toggle">
+    <details open>
+      <summary>Tutorials</summary>
+      <ol>
+        <li><a href="/en-US/docs/WebAssembly/Concepts">WebAssembly concepts</a></li>
+        <li><a href="/en-US/docs/WebAssembly/C_to_wasm">Compiling from C/C++ to WebAssembly</a></li>
+        <li><a href="/en-US/docs/WebAssembly/Using_the_JavaScript_API">Using the WebAssembly JavaScript API</a></li>
+        <li><a href="/en-US/docs/WebAssembly/Understanding_the_text_format">Understanding WebAssembly text format</a></li>
+        <li><a href="/en-US/docs/WebAssembly/Text_format_to_wasm">Converting WebAssembly text format to wasm</a></li>
+        <li><a href="/en-US/docs/WebAssembly/Loading_and_running">Loading and running WebAssembly code</a></li>
+        <li><a href="/en-US/docs/WebAssembly/Caching_modules">Caching compiled WebAssembly modules</a></li>
+        <li><a href="/en-US/docs/WebAssembly/Exported_functions">Exported WebAssembly functions</a></li>
```

Comment:
Not introduced here but @escattone, should these not be wrapped in `<%=text%>` tags?

Reply:
@schalkneethling no, this is the output of ``{{WebAssemblySidebar}}`` when rendered in ``en-US``.